prebid-server/endpoints/openrtb2/video.md

```diff
@@ -0,0 +1,348 @@
+
+```javascript
+durationrangesec = [15, 30] 
+```
+These values would indicate that ad durations of 15 ansd 30 seconds were allowed with this bid.
+
+If `podconfig.requireexactduration` is set to `false`, then the ads durations will be rounded up to the closest value in the `durationrangesec` array. Implicit minimum bid duration is 0.
+
+Using the values above, a ten second bid would be rounded to 15 seconds and an eighteen second bid would be rounded to 30 seconds.
+
+If `podconfig.requireexactduration` is set to `true`, then only the ads that are the exact duration match to the `durationrangesec` array will be allowed.
+
+Again, using the values above, a 15 second bid will be allowed  whereas an 18 second bid will be rejected.
+
+### Pods
+These are the parameters for the `pod` subobject:  
+
+{: .table .table-bordered .table-striped }
+| Param | Scope | Type | Description |
+| --- | --- | --- | --- |
+| podconfig.pods | Required | `Object` |  Container object describing the adPod(s) to be requested. |
+| pod.podid | Required | `Integer` |  Unique id of the pod within a particular request.   <br>**Recommendation**: Order these numerically, with increment of 1. |
+| pod.adpoddurationsec | Required | `Integer` |  Duration of the adPod. |
+| pod.configid | Required | `Integer` |  ID of the stored config that corresponds to a single pod request for all included adapters. |
+
+
+### Video Protocols
+A streaming protocol is a standardized method for delivering multimedia (usually video or audio) over the internet. It defines a method for sending "chunks" of content from one device to another and subsequently the method for reassembling those "chunks" into playable content.
+
+To indicate which protocol or protocols you wish to use for content management, pass an array of integers, each integer representing a video protocol. The table below list the integer/video protocol relationship. 
+
+{: .table .table-bordered .table-striped }
+| Integer | Protocol  |
+| --- | --- | 
+| 1  | VAST 1.0 |
+| 2  | VAST 2.0 |
+| 3  | VAST 3.0 |
+| 4  | VAST 1.0 Wrapper |
+| 5  | VAST 2.0 Wrapper |
+| 6  | VAST 3.0 Wrapper |
+| 7  | VAST 4.0 |
+| 8  | VAST 4.0 Wrapper |
+| 9  | DAAST 1.0 |
+| 10  | DAAST 1.0 Wrapper |
+
+### Buyer UIDs
+The `user.buyeruids` is an optional, but recommended parameter. It is an object that contains all the SSP UserIDs to send to the SSPs endpoint. They are passed as key-value pairs: `{bidder_name:bidder_specific_userid}`. 
+
+In the following list are the supported (registered) bidder names.  
+- "beachfront"
+- "gumgum"
+- "33across"
+- "adform"
+- "grid"
+- "rubicon"
+- "adtelligent"  
+- "brightroll"
+- "conversant"
+- "ix"
+- "lifestreet"
+- "pulsepoint"
+- "rhythmone"
+- "sonobi"  
+- "adkernelAdn"  
+- "audienceNetwork"  
+- "yieldmo"  
+- "openx"  
+- "pubmatic"  
+- "somoaudience"  
+- "sovrn"  
+- "appnexus"  
+- "eplanning"
+
```

Comment:
should the list be alphabetized? Or is there a reason for this order?

Reply:
Alphabitzed.